CHANGELOG.md

```diff
@@ -95,6 +95,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The plugin will no longer be loaded before the worlds are loaded
 - Citizens Holograms are now more robust on reload and reload faster
 - Added player death/respawn behavior to Region Objective and improved performance
+- Renamed `MAYOR` and `MAYOR_DEV` update strategies to `MAJOR` and `MAJOR_DEV`.
```

Comment:
No changelog entry is necessary, because the update was introduced in this version of BQ